server/src/main/java/com/strategists/game/controller/PlayerController.java

```diff
@@ -44,7 +44,7 @@ public List<Player> getPlayers() {
 	@PostMapping
 	public Player addPlayer(@RequestBody Map<String, Object> body) {
 		Assert.state(gameService.isLobbyState(), "Can't add players to active game!");
-		return playerService.addPlayer((String) body.get(Keys.USERNAME), (double) body.get(Keys.CASH));
```

Comment:
Why this change?

Reply:
The object that we were receiving had cash as Integer type and java can't convert Integer to double